autoblocks/_impl/tracer.py

```diff
@@ -76,6 +95,27 @@ def update_properties(self, properties: Dict) -> None:
         """
         self._properties.update(properties)
 
+    @contextmanager
+    def start_span(self):
```

Comment:
note this will only work properly in synchronous contexts. if a user has an async app we'll need to add an async tracer and use [contextvars](https://docs.python.org/3/library/contextvars.html) to have task-level properties